frontend/src/components/review/ExistingComment.tsx

```diff
@@ -68,7 +68,12 @@ const ExistingComment = ({
         }
       }
     } catch (err) {
-      handleError(err ?? "Error occurred, please try again.");
```

Comment:
is this left over correction from the old days of inconsistent linting :0

Reply:
i think so--i was getting an error when i was building locally that this wasn't linted properly, but the main firebase builds are passing hmm